src/backend/distributed/sql/udfs/citus_unmark_object_distributed/12.2-1.sql

```diff
@@ -0,0 +1,7 @@
+CREATE FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int, checkobjectexistence boolean)
+    RETURNS void
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$citus_unmark_object_distributed$$;
+COMMENT ON FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int, checkobjectexistence boolean)
+    IS 'Removes an object from citus.pg_dist_object after deletion. This version allows checking if the object exists before deletion.
+    If checkobjectexistence is true, object existence check performed. Otherwise, object existence check is skipped.';
```

Comment:
Given that we won't have the version that doesn't know about `checkobjectexistence` param after Citus upgrade, let's simplify the comment.

```suggestion
    IS 'Removes an object from citus.pg_dist_object after deletion. checkobjectexistence allows throwing an error if the object exists before deletion.';
```

Reply:
This function will have multiple versions. Therefore, I think we need to underline that checkobjectexistence parameter is being used by this version

Reply:
Added Default for checkobjectexistence and now there is single version